proxy-identity/main.go

```diff
@@ -75,30 +91,34 @@ func loadVerifier(pem string) (verify x509.VerifyOptions, err error) {
 //
 // If the key, CSR, and/or Crt paths refer to existing files, it is assumed that
 // the proxy has been restarted and these credentials are NOT recreated.
-func checkEndEntityDir(dir string) (string, string, error) {
+func checkEndEntityDir(dir string) (string, bool, string, bool, error) {
 	if dir == "" {
-		return "", "", errors.New("no end entity directory specified")
+		return "", false, "", false, errors.New("no end entity directory specified")
 	}
 
 	s, err := os.Stat(dir)
 	if err != nil {
-		return "", "", err
+		return "", false, "", false, err
 	}
 	if !s.IsDir() {
-		return "", "", fmt.Errorf("not a directory: %s", dir)
+		return "", false, "", false, fmt.Errorf("not a directory: %s", dir)
 	}
 
 	keyPath := filepath.Join(dir, "key.p8")
+	keyExists := false
 	if err = checkNotExists(keyPath); err != nil {
 		log.Infof("Using with pre-existing key: %s", keyPath)
+		keyExists = true
 	}
 
 	csrPath := filepath.Join(dir, "csr.der")
+	csrExists := false
 	if err = checkNotExists(csrPath); err != nil {
-		log.Infof("Using with pre-existing CSR: %s", keyPath)
```

Comment:
I know you're just copying the above log line... but the word _with_ in these messages breaks my brain ;p

Reply:
Ha no you're totally right I removed it!